topics/xml/index.md

```diff
@@ -0,0 +1,11 @@
+---
+created_by: World Wide Web Consortium
+display_name: XML
+released: 1996
+short_description: XML is subset of SGML (standard general markup language) used to store and transport data.
+topic: xml
+related: css, html, rss, xhtml
+url: http://www.w3.org/TR/rec-xml
```

Comment:
Not critical, but the link here redirects to `https://www.w3.org/TR/REC-xml/` which is https and has a different case. That said, this should probably be shortened to `https://www.w3.org/TR/xml/` as that is the "latest" rec link.